src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Sorry, but i didn't saw any message saying that it doesn't support 10-bit in your last log. But if you say so, we can stop the tests here.

At least it worked for one user without any optimization. Not bad.
Perhaps he have AMD :wink: